README.md

```diff
@@ -97,8 +97,8 @@ Below are instructions on installing and configuring a virtual machine to genera
 
 ## Setup Imagefactory:
 
-  * Clone imagefactory as /build/imagefactory (requires version 1.1.9-3 or
-    later for command line parameter passing and hyperv target support):
+  * Clone imagefactory as /build/imagefactory (requires git tag f13dc65
+    or later for google compute engine support):
```

Comment:
The GCE support also has the same symlink problem, just like HyperV.  Can you please add a note about manually creating symlink for GCE?